### ISSUE TITLE
I have a confusion about the details of the paper?

### ISSUE THREAD

**Issue by muzishen** [ifzhang/FairMOT#56]
From Table 2,  ResNet-50 means resnet50+fpn+dcn， and samely all model default have dcn?
Is my understanding correct？ Thank you ！

**Comment by ifzhang** [comment]
For ResNet, like CenterNet, we use dcn when upsampling. For ResNet-FPN, also we use dcn when we do upsample and concat. All models default have dcn. ResNet-34 means resnet34+dcn, ResNet-50 means resnet50+dcn and ResNet-34-FPN means resnet34+dcn+fpn.

**Comment by muzishen** [comment]
Thank you. I find that you train all data 30 epoch in the paper, but you provide 10 epoch model.  And batchsize should set 32 according to default opts.py？

**Comment by ifzhang** [comment]
The DLA-34 baseline model I provided is trained on large data for 30 epochs and finetuned on MOT17 for 10 epochs. The performance is 0.2 MOTA higher than training 30 epochs on large dataset. It achieves 68.7 MOTA on MOT16 test set and 67.5 MOTA on MOT17 test set. I think it's the best model I have trained.

**Comment by muzishen** [comment]
Q1：When you train all datesets,  How to differentiate the detection branch and  identity embedding branch according to datasets.  
Q2： How is the code divided into 2 branches to deal with different datasets. Because i found that the detected data set labels are all -1, will this affect the model if you train all dataset?

**Comment by ifzhang** [comment]
All the datasets use the two branches. In the identity embedding branch, the cross entropy loss will ignore -1 when doing id classifiaction.

**Comment by muzishen** [comment]
Why do you have training all the datasets (contain MOT17 ), you still fintune model on MOT17? Is it a trick?

**Comment by ifzhang** [comment]
Just a small trick +0.2 MOTA on MOT17 test set. It has little effect.
